device-services/device-random/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/edgexfoundry/device-random
+
+require (
+	github.com/edgexfoundry/device-sdk-go/v2 v2.0.0-dev.77
+	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.0.0-dev.100
```

Comment:
update both device-sdk-go and go-mod-core-contracts to v2.0.0

Reply:
Updated.